query/src/main/scala/filodb/query/exec/AggrOverRangeVectors.scala

```diff
@@ -33,7 +33,8 @@ final case class ReduceAggregateExec(queryContext: QueryContext,
     val task = for { schema <- firstSchema }
                yield {
                  val aggregator = RowAggregator(aggrOp, aggrParams, schema)
-                 RangeVectorAggregator.mapReduce(aggregator, skipMapPhase = true, results, rv => rv.key)
+                 RangeVectorAggregator.mapReduce(aggregator, skipMapPhase = true, results, rv => rv.key,
+                   querySession.qContext.groupByCardLimit)
```

Comment:
Can you pls confirm that this code runs during the monix query pipeline setup stage, and not during scan stage?